index.js

```diff
@@ -256,26 +256,23 @@ function getAPICredentials(cb) {
  *     to handle the auth response. False if manual entry used.
  */
  function authorize(useLocalhost) {
+  let codes = oauth2Client.generateCodeVerifier();
   let opts = {
     access_type: 'offline',
     scope: [
       'https://www.googleapis.com/auth/script.deployments',
       'https://www.googleapis.com/auth/script.projects',
     ],
-  }
+    code_challenge_method: 'S256',
```

Comment:
Please include a link to:
https://developers.google.com/identity/protocols/OAuth2InstalledApp#step1-code-verifier

Reply:
done